gnomad_qc/v3/annotations/generate_sv_hists.py

```diff
@@ -0,0 +1,197 @@
+# noqa: D100
+import argparse
+import logging
+
+import hail as hl
+from gnomad.utils.slack import slack_notifications
+
+from gnomad_qc.slack_creds import slack_token
+from gnomad_qc.v3.resources.annotations import sv_age_and_gq_hists
+from gnomad_qc.v3.resources.basics import (
+    gnomad_sv_autosome_vcf_paths,
+    gnomad_sv_release_samples_list_path,
+    gnomad_sv_sex_vcf_paths,
+    temp_gnomad_sv_mt_path,
+)
+from gnomad_qc.v3.resources.meta import meta
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s: %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("generate_gnomad_sv_histograms")
+logger.setLevel(logging.INFO)
+
+
+def generate_hists(mt: hl.MatrixTable) -> hl.Table:
```

Comment:
will we need to use this function again? do you think we should add this to gnomad_methods?

Reply:
asking the SV team to add this or keeping this in gnomad_qc sounds good to me